infrastructure/main.tf

```diff
@@ -6,19 +6,17 @@ locals {
   vaultName = "${var.product}-${var.env}"
 }
 
-data "azurerm_subnet" "core_infra_redis_subnet" {
-  name                 = "core-infra-subnet-1-${var.env}"
-  virtual_network_name = "core-infra-vnet-${var.env}"
-  resource_group_name = "core-infra-${var.env}"
-}
 
 module "probate-frontend-redis-cache" {
   source   = "git@github.com:hmcts/cnp-module-redis?ref=master"
   product  = "${var.product}-${var.component}-redis-cache"
   location = var.location
   env      = var.env
-  subnetid = data.azurerm_subnet.core_infra_redis_subnet.id
   common_tags  = var.common_tags
+  private_endpoint_enabled = true
+  redis_version = "6"
+  business_area = "cft"
+  public_network_access_enabled = false
 }
 
```

Comment:
On the sample pr they are setting max memory on variables, did you check that's set suitably in whatever base we are using?

Reply:
using the default one for the memory, the sample pr change the default which compatible with the Redis 6 (maxmemory-reserved and maxfragmentationmemory-reserved must be between 10% (642) and 60% (3855) of the cache's total memory.)